### PR TITLE
Add wire component

### PR DESCRIPTION
**Pull request by brianpeiris**

### CODE REVIEW

File: dist/aframe-altspace-component.js [hunk 1]
```diff
@@ -1207,5 +1208,30 @@
...
+/***/ },
+/* 2 */
+/***/ function(module, exports) {
+
+	AFRAME.registerComponent('wire',
+	{
+		schema: {
+			on: {type: 'string'},
+			emit: {type: 'string'},
+			els: {type: 'selectorAll'}
```
Comment (GavanWilhite) [inline dist/aframe-altspace-component.js]: targets? Makes sense in retrospect but took me a couple seconds to catch the name.

Comment (GavanWilhite) [reply dist/aframe-altspace-component.js]: might be nice to add `gained` / `lost` for input and `gain` / `lose` as output allow for states as well as events

Comment (GavanWilhite) [reply dist/aframe-altspace-component.js]: may also want to add support for multiple wire components, should be easy iirc

**Comment by stevenvergenz** [comment]
👍 LGTM

**Comment by stevenvergenz** [comment]
Since this component doesn't use any native features, I recommend rebasing this on master.

**Comment by brianpeiris** [comment]
@stevenvergenz True, but I assume we're going to release most of this sooner rather than later.

File: dist/aframe-altspace-component.js (hunk 1 above)
Comment (brianpeiris) [reply dist/aframe-altspace-component.js]: :+1: thanks, all fixed.

**Pull request merged**